models/wiki.go

```diff
@@ -115,7 +77,7 @@ func (repo *Repository) LocalWikiPath() string {
 
 // UpdateLocalWiki makes sure the local copy of repository wiki is up-to-date.
 func (repo *Repository) UpdateLocalWiki() error {
-	return updateLocalCopy(repo.WikiPath(), repo.LocalWikiPath())
+	return updateLocalCopy(repo.WikiPath(), repo.LocalWikiPath(), "")
```

Comment:
I at one time had this as "master" for the wiki, but it wasn't working, found I couldn't specify a branch. Not sure why. If you know how we can force the wiki to have a "master" branch, please let me know.